Content.Shared/SubFloor/SubFloorHideSystem.cs

```diff
@@ -152,10 +168,28 @@ private void UpdateEntity(EntityUid uid)
             }
 
             // Update normally.
-            UpdateEntity(uid, IsSubFloor(grid, grid.TileIndicesFor(transform.Coordinates)));
+            bool isSubFloor = IsSubFloor(grid, grid.TileIndicesFor(transform.Coordinates));
+            UpdateEntity(uid, isSubFloor);
+        }
+
+        // Toggles an enumerable set of entities to display.
+        public void ToggleSubfloorEntities(IEnumerable<EntityUid> entities, bool visible, EntityUid? uid = null, IEnumerable<object>? appearanceKeys = null)
+        {
+            foreach (var entity in entities)
+            {
+                if (!EntityManager.HasComponent<SubFloorHideComponent>(entity))
+                    continue;
+
+                var transform = EntityManager.GetComponent<TransformComponent>(entity);
+                if (_mapManager.TryGetGrid(transform.GridID, out var grid))
+                {
+                    bool isSubFloor = IsSubFloor(grid, grid.TileIndicesFor(transform.Coordinates));
```

Comment:
```suggestion
```
unused

Reply:
actually is the whole `transform` & `TryGetGrid` even needed there?

Reply:
good point - i think i left this in by accident

Reply:
You still left the `var transform = ...` in